### ISSUE TITLE
Docker for PHP Developers

### ISSUE THREAD

**Issue by adellibovi** [phpbarcelona/meetup-speakers#3]
Hi people,

Today I attend PHPBarcelona for the first time and I was thinking that I could add a little contribution.
I am an Italian, living in Barcelona for almost three years and working as full stack developer.
I am a promoter of Docker because of it's effectiveness, simplicity and reproducibility.

I was thinking that I could get ready a talk on how and why use Docker from developing/debug PHP apps to shipping to production.
Maybe for timing reason it would be good to just cover the developing/debug part, this also depends on how people attending are prepared, if they ever used or heard about Docker.

Also, I am not sure if this topic was never discussed since I couldn't find a list of previous topics, someone can point me to that?

Thanks,

**Comment by kbsali** [comment]
Hi Alfredo,

well that sounds like an awesome proposal!
Do you think you could get the presentation ready for the next meetup in +/- a month from now?
We already have 1 speaker you would be the 2nd one.

Thanks!

**Comment by adellibovi** [comment]
@kbsali Yeah I think I can :)

Since it was an idea that occurred me just today during the meet up, what we can do is: I will start to work on the presentation so I can get back to you in a week with a confirmation.
Obviously if someones else come with something ready I understand that you prioritise them. I guess that's not a problem neither since we could move this talk to another meetup.

Cheers,

**Comment by smoya** [comment]
Hi @adellibovi,

We are really happy with your proposal. However, We only have one free slot for May's meetup.
I'm gonna mark your talk proposal as "May proposal" and let other people send us other talks and vote them.

We will get back to you two weeks before the meetup. That works for you?

Cheers!

**Comment by adellibovi** [comment]
@smoya cool! That's perfect.

**Comment by smoya** [comment]
Hi @adellibovi ,

We confirm your talk for the next meetup (May's one). 
Still we have to confirm the date and the location of the venue, which we will do very soon.

Thanks

**Comment by smoya** [comment]
Hi @adellibovi ,

The meetup will take place on 10th May, at Skyscanner's Offices.
Anyway, could you please send, as soon as possible, your talk following this template:

```
Title

---
level: beginner | advanced | expert
twitter: your twitter handle
email: your email
---

Talk description
```

Thanks!

**Comment by adellibovi** [comment]
Hi @smoya,
I am flying back from London the 10th, what time will be the meetup?

**Comment by smoya** [comment]
7PM

**Comment by smoya** [comment]
@adellibovi  Is this timing fitting with your travel?

**Comment by adellibovi** [comment]
@smoya Yes, it fits :)
Can I make also a small demo in the talk? (Obviously fitting the 30 minutes limit)

**Comment by adellibovi** [comment]
```
Awesome Development Environments with Docker

---
level: beginner
twitter: @adellibovi
email: alfredo.dellibovi@gmail.com
---

Docker is a great tool that enables developers to build and test applications easily and fast.
We will understand what a container is, what benefits brings to PHP developers and why you should use it.
We will also have a demo on how to create a fully configured and set up like production container for your PHP application in less then 5 minutes.
```

**Comment by adellibovi** [comment]
@smoya  can you also correct my name in "Alfredo Delli Bovi" on the meetup description? It's actually one surname but with two words. :)

**Comment by smoya** [comment]
done @adellibovi

**Comment by adellibovi** [comment]
Hi @smoya I have a little inconvenient, my trip to London changed a bit and I must be there the 11th as well. Would be a big problem to find another one to replace my talk?

**Comment by kbsali** [comment]
ouch, that's a bit of a short notice :/
We have a couple of proposals, let's see if they can be if they can make it.

**Comment by adellibovi** [comment]
@kbsali yes, that's very unfortunate and I didn't knew until yesterday. Let me know when/if you find a solution

**Comment by vguardiola** [comment]
Hi Alfredo, we are planning a meetup on 17th November, You are interested in doing this talk on November meetup?

Thanks

**Comment by adellibovi** [comment]
@vguardiola sorry for my late reply, I missed the notification of your comment. Unlucky I won't be in Barcelona for mostly everyday of November :(

**Comment by dcorto** [comment]
@adellibovi lets see if we can plan a last meetup for December or maybe the first meetup of 2017 will be great have some Docker :)

**Comment by dcorto** [comment]
HI @adellibovi the next meetup is on 1st December, you are able to do this talk on this date?
